src/main/java/ama/validation/Validator.java

```diff
@@ -18,6 +21,11 @@
     private SubmissionPostRepository submissionPostRepository;
 
     public void validate(User user, BindingResult result) {
+        Pattern p = Pattern.compile("[^a-z0-9]", Pattern.CASE_INSENSITIVE);
```

Comment:
'^' is negation. source = https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html